crates/ruff_linter/resources/test/fixtures/pylint/bad_str_strip_call.py

```diff
@@ -64,10 +64,39 @@
 u''.lstrip('http://')
 
 # PLE1310
-b''.rstrip('http://')
+b''.rstrip(b'http://')
```

Comment:
Let's add a test that uses escaped backslashes

```suggestion
"\\test".strip("\\")
```